core/src/main/scala/org/scalarelational/column/property/NotNull.scala

```diff
@@ -1,8 +0,0 @@
-package org.scalarelational.column.property
-
-/**
- * @author Matt Hicks <matt@outr.com>
- */
-object NotNull extends ColumnProperty {
```

Comment:
This is already being taken care of by https://github.com/tindzk/scalarelational/commit/a4f4e1bae069f8d6ba1118ff90a497fe268813d6.